App.js

```diff
@@ -102,6 +103,19 @@ class App extends React.Component {
     i18n.setLocale(state.i18nReducer.locale);
   };
 
+  /*
```

Comment:
Please remove commented code so as to not cause confusion

Reply:
Ok.